### PR TITLE
OCPBUGS-14573: Agent-tui doesn't display full IPV6 addresses

### PR DESCRIPTION
**Pull request by pawanpinjarkar**

### CODE REVIEW

File: tools/agent_tui/ui/netstate_treeview.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ func (u *UI) ModalTreeView(netState net.NetState) (tview.Primitive, error) {
 	if err != nil {
 		return nil, err
 	}
-	width := 40
+	width := 45
```
[the remark this answers is not in the document]
Comment (pawanpinjarkar) [reply tools/agent_tui/ui/netstate_treeview.go]: Increased to `60` which seems sufficient.

**Review by rwsu — COMMENTED** [review]
/lgtm

**Pull request merged**